docs/griptape-tools/official-tools/google-gmail-client.md

```diff
@@ -26,42 +26,58 @@ gmail_tool = GoogleGmailClient(
 
 # Set up an agent using the GoogleGmailClient tool
 agent = Agent(
-    tools=[gmail_tool]
+    tools=[FileManager(), gmail_tool]
 )
 
 # Task: Create a draft email in GMail
 agent.run(
-    "Create a draft email in Gmail to example@email.com with the subject 'Test Draft', the body "
-    "'This is a test draft email.' and also add an attachment from my local machine with path "
-    "'/test/foo/bar.txt'.",
+    "I want sample1.txt and sample2.txt to be sent as attachments to my email to example@email.com "
```

Comment:
I think I prefer the original prompt phrasing, can we tweak that to include multiple attachments?

Reply:
Changed it back with multiple attachments